Modelica/Blocks/Logical.mo

```diff
@@ -20,6 +20,23 @@ the output is <strong>false</strong>.
 </html>"));
   end And;
 
+  block MultiAnd "Logical and of Boolean vector elements"
+    extends Modelica.Blocks.Interfaces.partialBooleanSO;
+    parameter Integer nin=2 "Number of inputs";
+    Modelica.Blocks.Interfaces.BooleanInput u[nin]
+      annotation (Placement(transformation(extent={{-140,-20},{-100,20}})));
+  equation
+    y=Modelica.Math.BooleanVectors.allTrue(u);
```

Comment:
Don't use the weird `allTrue`, use the strangely named but so much naturally defined `andTrue` instead:
```suggestion
    y = Modelica.Math.BooleanVectors.andTrue(u);
```